build_sdk.py

```diff
@@ -280,6 +280,21 @@ class ConfigInfo:
             "hello": Path("example/star64/hello")
         }
     ),
+    BoardInfo(
+        name="ariane",
+        arch=KernelArch.RISCV64,
+        gcc_cpu=None,
+        loader_link_address=0x90000000,
+        kernel_options={
+            "KernelIsMCS": True,
+            "KernelPlatform": "ariane",
+            "KernelRiscvExtD": True,
+            "KernelRiscvExtF": True,
```

Comment:
In the manual it says that Ariane has a `rv64i` CPU, that means we should disable compiling the FPU related extensions.

Reply:
(unless it's not `rv64i`?)

Reply:
It needs both FPU extensions